### ISSUE TITLE
Channel chat

### ISSUE THREAD

**Issue by RafaelGSS** [react-shop/api-pim#2]
Guys, I think that we need a chat to talk about our project.
What you think about gitter? Is open source and used for most of projects in github.

Ps: I already created [here](https:/gitter.im/ecommerce-shop), the name `react-shop` is not available :cry:. If anyone suggest other, I can remove.

**Comment by viniarruda** [comment]
Yeah, we need a chat for this. And I have a suggestion for task management, is the zepel.io. If you liked, tell me your e-mail to invite for project
